File: JavaScript/Recorrido_matriz_3D.js
const matriz3D = [
    [[1, 2], [3, 4]],
    [[5, 6], [7, 8]]
];

function recorrido3D(m) {
    let resultado = "";
    for (let capa of m)
        for (let fila of capa)
            for (let elemento of fila)
                resultado += elemento + " ";
    console.log(resultado);
}

recorrido3D(matriz3D);

File: Python/Recorrido_matriz_3D.py
matriz3D = [
    [[1, 2], [3, 4]],
    [[5, 6], [7, 8]]
]

def recorrido_3d(m):
    for capa in m:
        for fila in capa:
            for elemento in fila:
                print(elemento, end=" ")
    print()

recorrido_3d(matriz3D)